### DISCUSSION TITLE
Module demo JCB package

### DISCUSSION THREAD

**Discussion by laurentgarin** [joomengine/Joomla-Component-Builder#634] [Q&A] [answered]
Hello,

I am learning module in JCB, where i can download a demo of a module ?

Thanks.

**Comment by Llewellynvdm** [comment, marked as answer]
The current module structure is based on this structure https://github.com/vdm-io/boilerplate/tree/jcb/module

I do not have a demo module available but it is so [easy to build one](https://www.youtube.com/watch?v=lriWW3Uzyc8&list=PLQRGFI8XZ_wtGvPQZWBfDzzlERLQgpMRE&index=92), I would suggest adding on to the hello world or demo component, and then compile to see the outcome.

**Comment by laurentgarin** [comment]
OK, I did it, a few small questions, 
1 .it is not necessary to have a class helper ?
2. i have an array of objects so i use this code

`<?php
$vals = $data->getWebcasts();
foreach ($vals as $key => $item) {
		echo $item->title;		
}
?>`
Is it OK to do like that ?

Thanks.

**Reply by laurentgarin** [reply]
Sorry also, is it possible to use <?php echo $item->loadTemplate('iframefacebook'); ?> in module ?

Thanks.
